infra-as-code/bicep/modules/policy/assignments/alzDefaults/generateddocs/alzDefaultPolicyAssignments.bicep.md

```diff
@@ -182,6 +183,14 @@ Set Parameter to true to Opt-out of deployment telemetry
 
 - Default value: `False`
 
+### parPolicyEffect
+
+![Parameter Setting](https://img.shields.io/badge/parameter-optional-green?style=flat-square)
+
+Effect type for all policy definitions
```

Comment:
Likewise, this isn't the effect for all policies, just the sov baseline.

Reply:
Resolved to have sovereign policies in description.